jenkins/root-build.cmake

```diff
@@ -223,6 +230,12 @@ function(GET_ALL_SUPPORTED_MODULES_APPLE)
     builtin_xxhash
   )
 
+  if ("${ROOT_VERSION}" VERSION_GREATER "6.17")
```

Comment:
Why do we need this - even without we only have `root7` and `webgui` enabled if the options exist.